build/config.props

```diff
@@ -25,7 +25,7 @@
 
     <!-- Visual Studio Insertion Logic -->
     <VsTargetMajorVersion>$([MSBuild]::Add(11, $(MajorNuGetVersion)))</VsTargetMajorVersion>
-    <VsTargetBranch>main</VsTargetBranch>
+    <VsTargetBranch>d16.8</VsTargetBranch>
```

Comment:
I think you should change `IsEscrowMode`, rather than this property.

Reply:
Thanks for reviewing!
Fixed.